src/main/java/com/cburch/logisim/gui/menu/MenuEdit.java

```diff
@@ -220,9 +224,9 @@ private void populateRedoHistoryMenu() {
     } else {
       java.util.List<com.cburch.logisim.proj.Action> actions = proj.getRedoActions();
       for (final Action action : actions) {
-        JMenuItem actionItem = new JMenuItem(action.getName());
-        actionItem.addActionListener(new ActionListener() {
-        @Override
+          JMenuItem actionItem = new JMenuItem(action.getName());
+          actionItem.addActionListener(new ActionListener() {
+          @Override
```

Comment:
You indented the `@Override` as I requested, but you also incorrectly indented `JMenuItem...` and `actionItem... {` too far. This caused the linter to throw a bunch of indentation problems because the open curly was pushed over.